make/buf/scripts/release.bash

```diff
@@ -138,6 +140,20 @@ for os in Darwin Linux; do
   done
 done
 
+for os in Windows; do
+  for arch in x86_64 arm64; do
+    dir="${os}/${arch}/${BASE_NAME}"
+    # "${os}/${arch}"
+    zip_context_dir="$(dirname "${dir}")"
+    zip_dir="${BASE_NAME}"
+    zipfile="${BASE_NAME}-${os}-${arch}.zip"
+    pushd "${zip_context_dir}" >/dev/null
+    zip -r "${zipfile}" "${zip_dir}"
```

Comment:
I ran `SKIP_MINISIGN=1 make bufrelease` and it gives me the error that `zip` command is not found (in the [docker image](https://github.com/bufbuild/buf/blob/0b436b4cf494dd93cfc0ddb446d4fab5b07b514c/make/buf/all.mk#L157)). I also tried `golang:1.22-bullseye` and `golang:1.22-alpine` but neither worked.